scripts/app_manager

```diff
@@ -91,8 +91,11 @@ def main():
             rospy.logerr("Failed to load exchange: {}".format(e))
             sys.exit(1)
 
+    enable_overwrite = rospy.get_param('~enable_overwrite', True)
```

Comment:
I don't think the name `overwrite` makes sense here. You do not overwrite anything.
How about something like `can_replace_current_app` or `enable_app_replacement`?

Reply:
Thank you! I renamed to `enable_app_replacement`